### ISSUE TITLE
Support for PKG-INFO metadata format 2.1

### ISSUE THREAD

**Issue by kihneman** [enthought/okonomiyaki#310]
I tried to repack the latest cx_Oracle egg with edm and got the following error:
```
OkonomiyakiError: Unsupported PKG-INFO metadata format: u'2.1'
```

**Comment by RobinD42** [comment]
FYI, I've run into this issue this week as well. As a workaround I found that editing the metadata file in the source wheel and simply changing "2.1" --> "2.0" was accepted.

**Comment by tmcdan** [comment]
I'm also experiencing this issue when trying to repack [tensorflow_gpu](https://files.pythonhosted.org/packages/3d/21/63bf11a6bb4c1c85d52b259578bec20ef6f8894a13870f8f4f8b69635586/tensorflow_gpu-1.12.0-cp27-cp27mu-manylinux1_x86_64.whl) using
```
edm repack-wheel --extract-dependencies -e bootstrap -b 1 -p rh6-x86_64 tensorflow_gpu-1.12.0-cp27-cp27mu-manylinux1_x86_64.whl
```
Changing the version in the metadata file resulted in this error:
```
Unexpected error: OkonomiyakiError('No METADATA archive found in wheel file',)
```

**Comment by jdmarch** [comment]
>  simply changing "2.1" --> "2.0" was accepted.

@robin sanity check: why would 2.0 work if Okonomiyaki only supports up to 1.4?

**Comment by RobinD42** [comment]
> sanity check: why would 2.0 work if Okonomiyaki only supports up to 1.4?

I wondered that too when I found this issue, but I haven't dug further yet. I had just tried editing it in-place on a whim and it worked.

@tmcdan, I was also working on repacking tensorflow_gpu (for Windows) when I ran into the problem. How did you do the edit? I use MIdnight Commander and it is able to browse into zip files and edit files within the zip without needing to unzip/rezip them, so that's how I edited the wheels that needed modifications. However, for the Windows wheel the version in the MEDIDATA file is already set to "2.0" so I didn't actually need to modify that one. (Just some of the dependencies, IIRC.)
